Source/Request.swift

```diff
@@ -113,6 +113,9 @@ public class Request {
         var retryCount = 0
         /// Final `AFError` for the `Request`, whether from various internal Alamofire calls or as a result of a `task`.
         var error: AFError?
+        /// Whether the instance has had `finish()` called and is running the serializers. Should be replaced with a
+        /// representation in the state machine in the future.
+        var isFinishing = false
```

Comment:
I agree, this would be a good state change to make in the future. This will certainly work for now.